notebooks/packages.clj

```diff
@@ -0,0 +1,47 @@
+^{:clay {:kindly/options {:kinds-that-hide-code #{:kind/md :kind/hiccup :kind/html :kind/tex}}}}
+(ns packages
+  "A notebook demonstrating how to import and use Wolfram packages in Wolframite."
+  (:require
+   [wolframite.core :as wl]
+   [wolframite.wolfram :as w]
+   [wolframite.base.package :as wp]
+
+   [scicloj.kindly.v4.kind :as k]))
+
+^:kindly/hide-code
+(-> (scicloj.kindly-advice.v1.api/add-advisor! (fn [{:keys [value]}] (when (nil? value) [[:kind/hidden]])))
+    :k/hidden)
+
+(k/md "# How to work with packages
+
+Working with packages has never been easier! Following the Wolfram symbol convention, we introduce Wolframite's version of '<<', a.k.a 'Get', to both load the given context into the working kernel and to attach the Wolfram context's symbols (including functions) onto a clojure namespace: either using the context name as a symbol or the clojure symbol provided.
+
+The functions inside the  Wolfram package can then be used just like any other Wolfram functions.
+
+For example, try loading 'WolframPackage'. There are two functions defined inside:
+```wolfram
+  tryIt[x_] :=
+    Module[{y},
+      x^3
+    ]
+
+  additional[y_]:=3*y
+```
+")
+
+(wl/start)
+(wp/<< "resources/wolframPackage.wl" "WolframPackage")
+
+(wl/eval (WolframPackage/tryIt 10))
```

Comment:
I suppose this will make requiring this namespace impossible, b/c `WolframPackage/tryIt` is undefined at _read time_ - the same problem we fixed by introducing the wolrframite.wolfram ns. Right? Should we do st. about it / mention it?

Reply:
I'm not sure we need to 'require' it ? Let's chat about it. I'll make a separate issue.